src/app/registration/contact-form/contact-form.component.ts

```diff
@@ -24,10 +27,18 @@ export class ContactFormComponent implements OnInit {
       email: new FormControl(),
       license: new FormControl()
     });
+    this.myForm = this.fb.group(
+      {
+        firstName: ['', Validators.required],
+        lastName: ['', Validators.required],
+        email: ['', [Validators.required, Validators.email]],
+        phone: ['', [Validators.maxLength(10), Validators.minLength(10)]],
```

Comment:
can still submit outside of 10 digits because field is not required - is this limit necessary?

Reply:
de3434c